test/failures_testsuite/utils/failures.py

```diff
@@ -72,6 +73,43 @@ def minio_process_down(self, timeout):
                 continue
         return False
 
+    def zdb_process_down(self, count=1,timeout=100):
+        """
+        turn off zdb process , check it will be restart.
+        """
+        s3 = self._parent
+        if not s3:
+            return
+        n = 0
+        for namespace in s3.service.data['data']['namespaces']:
+            if n >= count:
+                break
+            robot = j.clients.zrobot.robots[namespace['node']]
+            robot = robot_god_token(robot)
+            ns = robot.services.get(name=namespace['name'])
+            zdb = robot.services.get(name=ns.data['data']['zerodb'])
+            try:
+                zdb.state.check('status', 'running', 'ok')
+                n +=1
+            except StateCheckError:
+                continue
+            logger.info('stop %s  process on node %s', zdb.name, namespace['node'])
+            zdb_node = j.clients.zos.get(zdb.name,data={"host": namespace['url'][7:-5]})
+            zdb_cont_client = zdb_node.containers.get("zerodb_{}".format(zdb.name))
+            zdb_node.client.container.terminate(zdb_cont_client.id)
```

Comment:
i'm not sure if they want the container down or the process .. so please check this .. Also anyways i think u need to make sure that the process is running inside the container not just that the container is up.